cosmos/settings.py

```diff
@@ -19,7 +19,7 @@
 cache_dir = Path(conf.get("cosmos", "cache_dir", fallback=DEFAULT_CACHE_DIR) or DEFAULT_CACHE_DIR)
 enable_cache = conf.getboolean("cosmos", "enable_cache", fallback=True)
 enable_dataset_alias = conf.getboolean("cosmos", "enable_dataset_alias", fallback=True)
-use_dataset_airflow3_uri_standard = conf.getboolean("cosmos", "enable_dataset_airflow3_uri", fallback=False)
+use_dataset_airflow3_uri_standard = conf.getboolean("cosmos", "use_dataset_airflow3_uri_standard", fallback=False)
```

Comment:
Thanks for raising this issue and trying to fix it, @Anti0ff! Unfortunately, this would be a breaking change. Is there any chance we could support both, for now?